usagov/vote-gov-drupal
# VOTE-2810, VOTE-2811, VOTE-2812 state content field changes

## Jira ticket

[VOTE-2810](https://cm-jira.usa.gov/browse/VOTE-2810) - New field type
[VOTE-2811](https://cm-jira.usa.gov/browse/VOTE-2811) - New block type
[VOTE-2812](https://cm-jira.usa.gov/browse/VOTE-2812) - Update State Content type

## Description

Add a new custom compound field. Add new fields to the State content type using the new compound field. Create a new block type using the new compound field along with other field types.

## Deployment and testing

### Post-deploy steps

1. Run 'lando retune' locally

### QA/Testing instructions

1. Login as an admin
2. Goto content admin page and edit a state content item
3. Locate the new tab, "State display content"
4. Confirm the available fields match the document located in https://cm-jira.usa.gov/browse/VOTE-2811
5. Add content to the fields and confirm that it saves
6. Edit the state again and confirm that the fields display in the fields
7. Goto content admin page again and go to block types
8. Locate the "State Display Content" link
9. Click the manage fields link and confirm the available fields match the document located in https://cm-jira.usa.gov/browse/VOTE-2812
10. Go back to the block types page and click the button up top to create a new block
11. Add a "State Display Content" block type
12. Fill out the form, save, and confirm that the block saves successfully
13. Edit the block and confirm that the fields display

## Checklist for the Developer




## Checklist for the Peer Reviewers
